packages/framework/data-object-base/api-report/data-object-base.api.md

```diff
@@ -5,75 +5,12 @@
-import { IFluidHandle } from '@fluidframework/core-interfaces';
-import { IFluidLoadable } from '@fluidframework/core-interfaces';
-import { IProvideFluidHandle } from '@fluidframework/core-interfaces';
-import { IRequest } from '@fluidframework/core-interfaces';
-import { IResponse } from '@fluidframework/core-interfaces';
-import { ISharedObject } from '@fluidframework/shared-object-base';
-import { ISharedObjectRegistry } from '@fluidframework/datastore';
 import { RuntimeFactoryHelper } from '@fluidframework/runtime-utils';
 import { RuntimeRequestHandler } from '@fluidframework/request-handler';
-import { TypedEventEmitter } from '@fluid-internal/client-utils';
-
-// @internal @deprecated (undocumented)
-export abstract class LazyLoadedDataObject<TRoot extends ISharedObject = ISharedObject, TEvents extends IEvent = IEvent> extends TypedEventEmitter<TEvents> implements IFluidLoadable, IProvideFluidHandle, IDisposable {
-    constructor(context: IFluidDataStoreContext, runtime: IFluidDataStoreRuntime, root: ISharedObject);
-    // (undocumented)
-    protected readonly context: IFluidDataStoreContext;
-    // (undocumented)
-    abstract create(props?: any): any;
-    // (undocumented)
-    dispose(): void;
-    // (undocumented)
-    get disposed(): boolean;
-    // (undocumented)
-    get handle(): IFluidHandle<this>;
-    // (undocumented)
-    get IFluidHandle(): IFluidHandle<this>;
-    // (undocumented)
-    get IFluidLoadable(): this;
-    // (undocumented)
-    get IProvideFluidHandle(): this;
-    // (undocumented)
-    abstract load(context: IFluidDataStoreContext, runtime: IFluidDataStoreRuntime, existing: boolean): Promise<void>;
-    // (undocumented)
-    request(r: IRequest): Promise<IResponse>;
-    // (undocumented)
-    protected readonly root: TRoot;
-    // (undocumented)
-    protected readonly runtime: IFluidDataStoreRuntime;
-}
-
-// @internal @deprecated (undocumented)
-export class LazyLoadedDataObjectFactory<T extends LazyLoadedDataObject> implements IFluidDataStoreFactory {
-    constructor(type: string, ctor: new (context: IFluidDataStoreContext, runtime: IFluidDataStoreRuntime, root: ISharedObject) => T, root: IChannelFactory, sharedObjects?: readonly IChannelFactory[], storeFactories?: readonly IFluidDataStoreFactory[]);
-    // (undocumented)
-    create(parentContext: IFluidDataStoreContext, props?: any): Promise<FluidObject>;
-    // (undocumented)
-    get IFluidDataStoreFactory(): this;
-    // (undocumented)
-    readonly IFluidDataStoreRegistry: IFluidDataStoreRegistry | undefined;
-    // (undocumented)
-    instantiateDataStore(context: IFluidDataStoreContext, existing: boolean): Promise<FluidDataStoreRuntime>;
-    // (undocumented)
-    readonly ISharedObjectRegistry: ISharedObjectRegistry;
-    // (undocumented)
-    readonly root: IChannelFactory;
-    // (undocumented)
-    readonly type: string;
-}
 
 // @internal (undocumented)
 export class RuntimeFactory extends RuntimeFactoryHelper {
```

Comment:
where are these used? can we move them somewhere else and just delete the whole package?

Reply:
The Lazy items aren't used anywhere anymore after #19482 - I targeted them because they exhibited some patterns I actively want to discourage (e.g. EventForwarder).

I don't believe RuntimeFactory is used by anyone either, but I don't have an "anti" stance on it in the same way to motivate it's elimination currently.  I mentioned in #19438 that I think its fate is either to get good and move to Aqueduct or else we might just get rid of it, but I haven't developed an opinion on it yet.